.github/workflows/upload_all.yml

```diff
@@ -1,26 +1,133 @@
-      POLYPOD_ANDROID_UPLOAD_KEYSTORE_PASS: ${{ secrets.POLYPOD_ANDROID_UPLOAD_KEYSTORE_PASS }}
-      POLYPOD_ANDROID_UPLOAD_KEY_ALIAS: ${{ secrets.POLYPOD_ANDROID_UPLOAD_KEY_ALIAS }}
-      POLYPOD_ANDROID_STORE_SERVICE_ACCOUNT_JSON: ${{ secrets.POLYPOD_ANDROID_STORE_SERVICE_ACCOUNT_JSON }}
+    if: contains(fromJson('["all", "google-play"]'), github.event.inputs.targets)
+    needs: check_preconditions
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 16
+          cache: "npm"
+          cache-dependency-path: "**/package-lock.json"
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: API and features - Build
+        env:
+          POLYPOD_POLYPEDIA_REPORT_URL: ${{ secrets.POLYPOD_POLYPEDIA_REPORT_URL }}
+          POLYPOD_POLYPEDIA_REPORT_AUTHORIZATION: ${{ secrets.POLYPOD_POLYPEDIA_REPORT_AUTHORIZATION }}
+        run: ./build.js
+      - name: Android - Test
+        working-directory: platform/android
+        run: ./gradlew test
+      - name: Android - Write keystore file
+        working-directory: platform/android/app
+        env:
+          KEYSTORE: ${{ secrets.POLYPOD_ANDROID_UPLOAD_KEYSTORE_FILE }}
+        run: 'echo -n "$KEYSTORE" | base64 --decode > keystore.tmp'
+      - name: Android - Build
+        working-directory: platform/android
+        env:
+          POLYPOD_ANDROID_UPLOAD_KEYSTORE_FILE: keystore.tmp
+          POLYPOD_ANDROID_UPLOAD_KEYSTORE_PASS: ${{ secrets.POLYPOD_ANDROID_UPLOAD_KEYSTORE_PASS }}
+          POLYPOD_ANDROID_UPLOAD_KEY_ALIAS: ${{ secrets.POLYPOD_ANDROID_UPLOAD_KEY_ALIAS }}
+        run: ./gradlew assemble -Psign=true
+      - name: Determine release name
+        working-directory: platform/android
+        run: |
+          aapt_path=/usr/local/lib/android/sdk/build-tools/30.0.0/aapt
+          apk_path=./app/build/outputs/apk/release/app-release.apk
+          version_code=$(./getPackageInfo.sh $aapt_path $apk_path versionCode)
+          version_name=$(./getPackageInfo.sh $aapt_path $apk_path versionName)
+          branch=${GITHUB_REF#refs/heads/}
+          sha_short=${GITHUB_SHA::7}
+          release_name="$version_code ($version_name) - $branch@$sha_short"
+          echo ::set-output name=release_name::$release_name
+      - name: Upload
+        # uses: r0adkll/upload-google-play@9745ef904e395471bca5696056a6ce8a60d18cf8
+        uses: polypoly-eu/upload-google-play@master
+        with:
+          serviceAccountJsonPlainText: ${{ secrets.POLYPOD_ANDROID_STORE_SERVICE_ACCOUNT_JSON }}
+          packageName: coop.polypoly.polypod
+          releaseName: ${{ steps.vars.outputs.release_name }}
+          releaseFiles: ./platform/android/app/build/outputs/apk/release/app-release.apk
+          track: internal
+          # whatsNewDirectory: distribution/whatsnew
```

Comment:
Just delete this if you're not using it.

Reply:
Left it from the original workflow - guess we could indeed, but then again we're on the verge of adding non-draft publishing back, so if you don't mind too much I'll leave it for now, to finally merge this PR without having to do any further testing :D